spec/spec_helper.rb

```diff
@@ -21,6 +21,13 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 RSpec.configure do |config|
+  if ENV['CI']
+    config.before(:example, :focus) { raise "Should not commit focused specs" }
+  else
+    config.filter_run focus: true
+    config.run_all_when_everything_filtered = true
+  end
+
```

Comment:
I'll add a comment! It allows you to filter which specs you run which is helpful when you want to isolate one specific test case. This might help: https://www.relishapp.com/rspec/rspec-core/v/2-0/docs/filtering/inclusion-filters